proposals/awslogs.md

```diff
@@ -0,0 +1,195 @@
+# ECS CLI Logging
+
+## Overview
+
+The following Proposal lays out a design and implementation plan for creating a user experience in the ECS CLI for getting container logs from CloudWatch.
+
+### Use Cases
+
+1. User has a known error and wants to find more info on it.
+2. User is doing a deployment and wants to tail the logs and grep for errors.
+3. User wants to quickly set up their CloudWatch Logs based upon the configuration specified in their docker compose file, and have the CLI creates any necessary log groups for them.
+4. User wants to monitor their task/service, so they continually stream the logs.
+
+## Phase 1 Solution
+Top level `ecs-cli logs` command that will not use the docker compose file. This allows it to be used by a wide array of ECS customers, not just compose users. The command will allow customers to find logs for a given task.
+
+```
+ecs-cli logs --help
+--follow                         Stream logs (continuously poll for updates)
```

Comment:
Why not call this "stream"? seems more consistent with usual logging terminology (and also seems to be what CW Logs uses)

Reply:
`--follow` is used by `tail(1)` and `docker(1)`.

Reply:
Yup, that's the reason why we chose to call it `--follow`